SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -556,7 +556,7 @@ SCDocHTMLRenderer {
 				if(noParBreak) {
 					noParBreak = false;
 				} {
-					stream << "\n<p>";
+					stream << "\n";
```

Comment:
can you try to use a br to restore the linebreaks, so

```suggestion
					stream << "<br />\n";
```

Unclosed `<p>` are actually invalid syntax and it is up to the browser to recover from it - it seems the browser close `<p>` on the next HTML tag in this case, creating an empty paragraph which also is an implicit linebreak.

Reply:
Thanks!